src/python/WMComponent/DBS3Buffer/MySQL/Create.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, logger=None, dbi=None, params=None):
                  subscribed             INTEGER      DEFAULT 0,
                  phedex_group           VARCHAR(100),
                  delete_blocks          INTEGER,
+                 dataset_lifetime       INTEGER     DEFAULT 46656000,
```

Comment:
Jhonatan, if you are actually setting `dataset_lifetime` for every dataset from the RunConfigAPI, then having a default value of `0` looks more reasonable to me.
In the RucioInjector code, we could even have a check like:
```
if dataset_lifetime > 0:
  kwargs['lifetime'....
```
such that when it's 0, nothing happens. What do you think?

Reply:
Indeed, is more reasonable. I also modified RucioInjector in the clause of add that parameter